### PR TITLE
Enable k8s plugins for transform

### PR DESCRIPTION
**Pull request by eemcmullan**
Enable k8s plugins from crane-lib for transform. 
#22

### CODE REVIEW

**Review by sseago — CHANGES_REQUESTED** [review]
This looks good, but I'd suggest a minor refactoring. Currently we have the concept of both binary and internal plugins (although we only have one example of the latter right now). GetBinaryPlugins pulls from the plugin dir. I think I'd rather preserve the functionality of GetBinaryPlugins as it exists today. Perhaps a new func `GetPlugins(dir string)` which starts with the kubernetes plugin (a list with one item in it), and then appends the results of `GetBinaryPlugins` to that list. Basically putting the new code from this PR into this new func, which calls the existing func. Otherwise people who weren't involved in this refactoring might be confused that `GetBinaryPlugins` returns both binary and internal plugins.

**Comment by sseago** [comment]
Actually, with this refactoring, `GetBinaryPlugins` doesn't need to be an exported function, so forget what I said above. Leave it as-is, but just rename the exported func `GetPlugins` (and change the places where it's called).

**Comment by eemcmullan** [comment]
Just updated `GetBinaryPlugins` to `GetPlugins`.

**Review by sseago — APPROVED** [review]

File: cmd/transform/optionals/optionals.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ import (
 )
 
 type Options struct {
-	logger            logrus.FieldLogger
-	PluginDir         string
+	logger    logrus.FieldLogger
+	PluginDir string
```
Comment (shawn-hurley) [inline cmd/transform/optionals/optionals.go:14]: +1 thanks for fixing gofmt

**Review by shawn-hurley — APPROVED** [review]
Looks good to me +1

thanks!

**Pull request merged**